src/java/picard/illumina/ClusterDataToSamConverter.java

```diff
@@ -65,6 +67,10 @@
     private final int outputRecordsPerCluster;
     private final ReadNameEncoder readNameEncoder;
 
+    private String molecularIndexTag          = "RX";
```

Comment:
perhaps add a todo to change this to reference SAMTagsUtil when I finally get around to changing the samspec: https://github.com/samtools/hts-specs/pull/119